docs/respository-configuration-cli.md

```diff
@@ -0,0 +1,383 @@
+# rv configure repository - Command Examples
+
+## Overview
+
+The `rv configure repository` command allows you to:
+- Add repositories with precise positioning (`--first`, `--last`, `--before`, `--after`)
+- Replace existing repositories (`--replace`)
+- Remove specific repositories (`--remove`)
+- Clear all repositories (`--clear`)
+- Enable force source compilation (`--force-source`)
+- Output results in JSON format (`--json`) or detailed text format
+
+Both output formats provide comprehensive information including operation type, repository alias, and URL details.
+
+### Initial Configuration
+```toml
+[project]
+name = "demo-project"
+r_version = "4.4"
+repositories = [
+    {alias = "cran", url = "https://cran.r-project.org"},
+    {alias = "posit", url = "https://packagemanager.posit.co/cran/2024-12-16/"}
+]
+dependencies = [
+    "dplyr",
+    "ggplot2",
+]```
+
+## Example 1: Add repository as first entry
+
+**Command:**
+
+```bash
+./target/release/rv configure repository --config-file test-example/rproject.toml --alias "ppm-latest" --url "https://packagemanager.posit.co/cran/latest" --first
```

Comment:
let's use just rv for the doc example. How does that doc file overlap with the doc site since we will need to document things there as well